viz/tailwind.config.ts

```diff
@@ -202,11 +104,27 @@ const config: Config = {
     require("tailwindcss-animate"),
     require("@tailwindcss/container-queries"),
   ],
+  // We don't generate css based on generated contents so we try to include as many classes as possible. 
   safelist: [
     {
-      pattern: /./, // This matches all class names.
+     pattern: /./, // This includes all the utility classes.
+    },
+    {
```

Comment:
We can add more but the css file size is going to be huge if we add everything, let's start with this and add more if needed